pages/consulting/_how-we-work.html

```diff
@@ -0,0 +1,117 @@
+<div class="row row-page-header square-section" style="margin-bottom: 60px;">
+  <div class="col-xs-12">
+    <h2>How We Work</h2>
```

Comment:
A request: as you scroll, the previous text disappears completely, so it's easy to miss things and lose a bit of the narrative.

Could we make two tweaks:

1. Instead of setting the opacity of the "inactive" text to 0, could we set it to 10 or 20, so it is still faintly visible?
2. Note that currently, the text overlaps a bit, so to make this work, we'd also have to space it out vertically more.

Reply:
Note: if this change takes more than 30m, let's do it in a follow-up PR, as we don't need to block getting this merged & live for this change.

Reply:
Hi @brikis98 , Tola updated the design. Please have a look at it here: https://www.figma.com/file/sdZraPSRCb0Q13H3SzkGEM?node-id=337:1826#461208995 
and let us know which version you prefer.

Reply:
Saw that, thank you! As I wrote in Figma, let's go with option 1. Thanks!